Sources/Logging/Logging.swift

```diff
@@ -454,7 +494,10 @@ extension Logger {
     ///
     ///     logger.info("Hello \(world)")
     ///
-    public struct Message: ExpressibleByStringLiteral, Equatable, CustomStringConvertible, ExpressibleByStringInterpolation {
+    public struct Message: ExpressibleByStringLiteral,
+                           Equatable,
+                           CustomStringConvertible,
+                           ExpressibleByStringInterpolation {
```

Comment:
These lines are questionable. I'm not sure about this formatting convention.

Reply:
I'd be fine with this